packages/@n8n/nodes-langchain/nodes/llms/LMChatAnthropic/LmChatAnthropic.node.ts

```diff
@@ -105,7 +111,19 @@ export class LmChatAnthropic implements INodeType {
 				default: 'claude-3-sonnet-20240229',
 				displayOptions: {
 					hide: {
-						'@version': [1],
+						'@version': [{ _cnd: { gte: 1.2 } }, 1],
```

Comment:
not sure about this maybe instead show on version 1.1?

Reply:
Ah, of course 🤦 Updated, could you re-approve, please?